content/en/metrics/volume.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Volume
+description: "Understand and manage your custom metrics usage and costs with the Volume Management page."
+further_reading:
+  - link: "/metrics/summary/"
+    tag: "Documentation"
+    text: "Metrics Summary"
+  - link: "/metrics/metrics-without-limits/"
+    tag: "Documentation"
+    text: "Metrics without Limits™"
+  - link: "/metrics/custom_metrics/"
+    tag: "Documentation"
+    text: "Custom Metrics"
+
+---
+
+## Overview
+
+{{< img src="metrics/volume/metrics_volume_overview.png" alt="Metrics Volume page set to a timeframe of the past hour (by default) showing the search, filter, facet and column sorting features available" style="width:100%;" >}}
+
+Cloud-based applications generate massive amounts of data, which can be overwhelming for your organization as it scales. Observability costs become a significant budget item but core observability teams lack visibility into what is truly valuable to each individual engineering team. Individual teams are less incentivized to be proactive in helping manage this growth because they have limited insights into the costs of the metrics and tags they're submitting.
+
+Datadog's [Metrics Volume Management page][1] provides comprehensive visibility and intelligent insights for which metrics you should focus your cost-optimization efforts. When used with [Metrics without Limits™][3], Metrics Volume allows for flexible configuration of metrics ingestion and indexing to reduce costs without sacrificing accuracy. 
+
+With the Metrics Volume Management page you can quickly answer the following questions in real-time: 
+- What is my overall account's realtime estimated Indexed Custom Metrics usage?
+- What is my overall account's realtime estimated Ingested Custom Metrics usage?
+- What are the Top 500 largest Metrics without Limits configured metric names by Ingested Custom Metrics volume?
+- What are the Top 500 largest metric names by Indexed Custom Metrics volume?
+- What are the Top 500 spiking cardinality metric names?
```

Comment:
Can we preserve it as 3 different questions? here's why: 
* Was hopeful we can keep this summary section framed as questions that customers would ask. And wanted to follow the example external docs' (see gdoc for governance guide) where they call out each individual functionality to emphasize the huge value the tool brings. Even if people gloss over the summary section, they'll at least see numerically that the page offers a lot.
* Think it still makes sense to keep as separate bullets here because indexed vs ingested are separate SKUs. 
* conceptually my top 500 largest metric names by indexed volume are likely a different group of metrics and require a different user action to access than the top 500 spiking metric names.